commit 6ecb0918ced6a19c3fc992739a5b2afe13e43b34
Author: Jon Nettleton <jon@laptop.org>
Date:   Sun Jan 5 15:03:05 2014 +0100

    Revert "ENGR00288972 HDMI: unmask overflow interrupter when HDMI phy power on"
    
    This reverts commit ce224e3c4fc3c1b00a94641c28724be2adcd78be.

diff --git a/drivers/video/mxc/mxc_hdmi.c b/drivers/video/mxc/mxc_hdmi.c
index cbf6b85..ef63a97 100644
--- a/drivers/video/mxc/mxc_hdmi.c
+++ b/drivers/video/mxc/mxc_hdmi.c
@@ -1301,8 +1301,6 @@
 	}
 
 	hdmi->phy_enabled = true;
-	if (!hdmi->hdmi_data.video_mode.mDVI)
-		hdmi_enable_overflow_interrupts();
 }
 
 static void hdmi_config_AVI(struct mxc_hdmi *hdmi)
@@ -2262,6 +2255,9 @@ static void mxc_hdmi_setup(struct mxc_hdmi *hdmi, unsigned long event)
 
 	mxc_hdmi_clear_overflow(hdmi);
 
+	if (!hdmi->hdmi_data.video_mode.mDVI)
+		hdmi_enable_overflow_interrupts();
+
 	dev_dbg(&hdmi->pdev->dev, "%s exit\n\n", __func__);
 
 }
